### ISSUE TITLE
Is it possible to filter docker events

### ISSUE THREAD

**Issue by korya** [DataDog/dd-agent#1960]
Is it possible to filter particular docker events for some of the containers?

My use case. I'm running a cassandra node within a docker container, e.g. `cassandra-node`. Its discovery service is being executed once in 10 seconds within the same container `docker exec cassandra-node ...`.

Question: is there a way I can configure dd-agent to ignore docker `exec` events coming from `cassandra` container? I still need to receive other events (`start`, `stop`, etc.).

**Comment by remh** [comment]
@korya this is not possible for now. However you could create a filter in the event stream to match only the events you care about.

e.g. something like
```
sources:docker star OR create OR stop OR die 
```
https://app.datadoghq.com/event/stream?tags_execution=and&show_private=true&per_page=30&query=sources:docker%20status:all%20priority:all%20star%20OR%20create%20OR%20stop%20OR%20die&aggregate_up=true&use_date_happened=false&display_timeline=true&from_ts=1443020400000&live=true&is_zoomed=false&to_ts=1443625200000&is_auto=false&incident=true&only_discussed=false&no_user=false&page=0&bucket_size=10800000

would that work for you ?

**Comment by korya** [comment]
Not really. My main problem is the graphs showing events on my dashboards just become red.

<img width="508" alt="screen shot 2015-09-30 at 23 13 19" src="https://cloud.githubusercontent.com/assets/148461/10212137/f671c54c-67c8-11e5-8ad0-51899bc6db84.png">

I hoped to have a way to filter out these events before they are sent to datadog servers.
